spec/features/navigation/guest_user_can_visit_home_page_spec.rb

```diff
@@ -1,6 +1,11 @@
 require 'rails_helper'
 
 RSpec.feature "user can visit home page" do
+
+  before :each do
+    create(:project)
+  end
+
   context "as a guest user" do
```

Comment:
Our index page calls our first project, and if the project doesn't exist in the test DB, the call for `project.image_url` errors with a null value. (see other comment for more context)